build/build.tcl

```diff
@@ -72,17 +72,22 @@ proc ip_address {string} {
     format "%o" $x
 }
 
+# Respond to the output from (load ...).
+proc respond_load { r } {
+    expect -re {[\r\n][0-9]+\.? *[\r\n]}
+    type $r
+}
+
```

Comment:
That is *such* a good idea.  You don't know how many times that size/offset has changed and broken my build (attempts).

Reply:
It took several tries to come up with a Tcl regexp that works - I ended up writing an expect script to test it against ITS over telnet...